gradio/components/image.py

```diff
@@ -47,15 +45,16 @@ def __init__(
         self,
         value: str | _Image.Image | np.ndarray | None = None,
         *,
-        shape: tuple[int, int] | None = None,
         height: int | None = None,
         width: int | None = None,
         image_mode: Literal[
             "1", "L", "P", "RGB", "RGBA", "CMYK", "YCbCr", "LAB", "HSV", "I", "F"
         ] = "RGB",
-        invert_colors: bool = False,
-        source: Literal["upload", "webcam", "canvas"] = "upload",
-        tool: Literal["editor", "select", "sketch", "color-sketch"] | None = None,
+        sources: list[Literal["upload", "webcam", "clipboard"]] = [
+            "upload",
+            "webcam",
```

Comment:
Once we create the `ImageEditor` component, it'd be nice to add the `tool` argument back to `Image`, and if a person tries to create an `Image` with a tool, we display a warning and just create the `ImageEditor` component for them. I think it'll help with the migration

Reply:
Sounds good.